int16 scoopSpeed
int16 actuatorSpeed
bool toggleLights
bool moveSlurry
bool site1
bool site2
bool site3
bool site4
bool site5
bool clean1
bool clean2
bool clean3
bool predAddWater
bool purge
bool hardStop
